include/ametsuchi/io/cvfs.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <ametsuchi/io/vfs.h>
+#include <ametsuchi/pager/page.h>
+#include <ametsuchi/cache.h>
+#include <cstdio>
+
+namespace ametsuchi {
+namespace io {
+
+enum { CVFS_MAX_FILE_SIZE = 4096 * 16, CVFS_MAX_CACHE_SIZE = 256, };
```

Comment:
Why these are not in globals.h?
`include/ametsuchi/pager/page.h` contains page size, but you don't use it here (I assume that 4096 here is the page size) -- this is a reason why we need to move variables like this to globals.h